boards/shields/nxp_m2_wifi_bt/doc/index.rst

```diff
@@ -0,0 +1,56 @@
+.. _nxp_m2_wifi_bt:
+
+NXP M.2 Wi-Fi/BT Interface
+##########################
+
+Overview
+********
+
+This shield supports M.2 modules (Murata/ublox etc) to use Bluetooth HCI UART Interface and enable
+support for Bluetooth applications and enables SDIO interface for Wi-Fi applications. It supports
+M.2 modules for NXP Wireless-Controller SoC IW416 and IW612 which works for both WIFI and Bluetooth.
+
+More information about IW416 & IW612 Wireless Controller can be found at `NXP IW416`_ and `NXP IW612`_
+
+Requirements
+************
+
+To use the shield, below requirements needs to be satisfied.
```

Comment:
Is there a webpage with information about this shield board. It would be good to include that in this page.